docs/app/Examples/modules/Popup/Triggers/PopupClickExample.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Button, Icon, Popup } from 'stardust'
+
+export default function () {
+  return (
+    <Button.Group>
+      <Button color='red'>Activate doomsday device</Button>
+      <Popup
+        trigger={<Button color='black' icon><Icon name='question' /></Button>}
```

Comment:
The button can be simplified with shorthand syntax here.  Shorthand is very useful for cases exactly like this, passing components through props:

```jsx
trigger={<Button color='black' icon='question' />}
```

Then, you can also remove the `Icon` import.